packages/ui-extensions-dev-console/src/QRCodeModal/QRCodeModal.test.tsx

```diff
@@ -72,7 +72,7 @@ describe('QRCodeModal', () => {
     );
 
     expect(container?.find(QRCode)?.prop('value')).toStrictEqual(
-      `com.shopify.pos://pos-ui-extensions?url=${extension.development.root.url}`,
+      `com.shopify.pos://pos-ui-extensions?url=${extension.development.root.url}&appId=${app.id}`,
```

Comment:
I don't think this is safe to use - the only field that's guaranteed to be populated currently is the `app.apiKey`